### DISCUSSION TITLE
How to run atom with new version of electron

### DISCUSSION THREAD

**Discussion by maxsupermanhd** [atom/atom#22646] [Q&A] [answered]
I have very nasty #21825 issue, looks like in newer version of electron it is fixed. Is there a way to build atom with new electron and if so how can I do it?

**Comment by icecream17** [comment, marked as answer]
There no way right now :(

Electron status:

Currently there's a pr attempting to update to Electron 11.4.7: https://github.com/atom/atom/pull/22687
This pr allows updating to 10.4.0: https://github.com/atom/atom/pull/22125
The current electron version is 9.4.4: https://github.com/atom/atom/blob/master/package.json#L15

**Reply by maxsupermanhd** [reply]
I want this fix applied - should fix initial issue https://github.com/electron/electron/pull/29445
